circle.yml

```diff
@@ -19,12 +19,10 @@ machine:
   java:
     version: openjdk8
   post:
-    # Manually install very recent version of cassandra; unless we create a user we have to run it as root
-    # uncomment below when datastax has version 3.9 available. at time of commit, latest was 3.7
-    #- curl -SL http://www-us.apache.org/dist/cassandra/3.9/apache-cassandra-3.9-bin.tar.gz | tar xz
-    - curl -SL http://cassci.datastax.com/job/trunk/lastSuccessfulBuild/artifact/build/apache-cassandra-3.10-SNAPSHOT-bin.tar.gz | tar xz
-    - sed -i -e 's/INFO/WARN/g' apache-cassandra-*/conf/logback.xml
-    - sudo ./apache-cassandra-*/bin/cassandra -R
+    # Manually install very recent version of cassandra
+    - curl -SL http://downloads.datastax.com/community/dsc-cassandra-3.0.8-bin.tar.gz | tar xz
```

Comment:
@abesto not sure if this died because of the out-of-memory warning we are getting in circleci, or if I somehow made a syntax error I can't see.. can you give second eyes? https://circleci.com/gh/openzipkin/zipkin/138